### PR TITLE
update species_fig

### PR DESCRIPTION
**Pull request by lauterbur**

### CODE REVIEW

**Comment by petrelharp** [comment]
One more issue - CaeEle has a genetic map, not a demographic model.

**Comment by petrelharp** [comment]
Other than that it looks good!

File: figs/species_fig.R [hunk 1]
```diff
@@ -31,14 +31,12 @@ ggtree(tree, size=2) %<+% species +
...
-            c("Anas platyrhynchos",0,0,0,0),
+data<-rbind(c("Aedes aegypti",0,0,0,0),
+            c("Anas platyrhynchos",1,0,0,0),
             c("Anolis carolinensis",0,0,0,0),
             c("Anopheles gambiae",1,0,0,0),
             c("Apis mellifera",0,0,0,0),
-            c("Arabidopsis thaliana",3,1,2,0),
+            c("Arabidopsis thaliana",3,1,2,1),
             c("Bos taurus",1,0,0,0),
             c("Caenorhabditis elegans",1,0,0,0),
```
Comment (petrelharp) [inline figs/species_fig.R]:
```suggestion
            c("Caenorhabditis elegans",0,1,0,0),
```

Comment (petrelharp) [reply figs/species_fig.R]: need to update the pdf too though

**Comment by petrelharp** [comment]
I fixed this.

**Comment by lauterbur** [comment]
Sorry, didn't see it! Thanks for fixing CaeEle.

**Pull request merged**